cai_causal_graph/causal_graph.py

```diff
@@ -382,6 +382,14 @@ def __init__(
         self._skeleton = Skeleton(graph=self)
         self._sepsets: dict = dict()
 
+    @staticmethod
+    def _node_class():
+        return Node
```

Comment:
Type and docstring, for `_edge_class` below as well

Reply:
+1